src/app/volunteer/login/page.tsx

```diff
@@ -0,0 +1,23 @@
+import ErrorModal from '@/components/volunteer/ErrorModal/ErrorModal';
+import KakaoLogin from '@/components/volunteer/KakaoLoginButton/KakaoLogin';
+import MarqueeTitle from '@/components/volunteer/MarqeeTitle/MarqueeTitle';
+import { headers } from 'next/headers';
+
+export default function VolunteerLogin({}) {
+  const header = headers();
+  const referer = header.get('referer');
+
+  /**
+   * voluter/redirect에서 오는 경우 에러처리
+   * (isMember == true , accessToken && refreshToken == false 인 경우
+   */
+  const error = Boolean(referer && referer.match('volunteer/redirect'));
+
+  return (
+    <div>
+      <MarqueeTitle />
+      <KakaoLogin />
+      <ErrorModal error={error} />
```

Comment:
ErrorModal은 에러 상태일 때만 렌더링되는 컴포넌트입니다
error prop을 넘기는 대신에 조건부 렌더링을 이용하면 더 단순하고 직관적일 것 같아요!
```suggestion
      {error && <ErrorModal />
```
추가로 error가 무슨 에러인지 이름에서 나타내주면 좋을 것 같아요
error 대신 authError / redirectError / tokenError 등의 네이밍을 사용했으면 좋겠습니다 (모달 컴포넌트 이름도 마찬가지)

Reply:
이부분 미흡하게 마무리되어 정상적으로 동작하지 않는 코드인데,,,,, 수정해서 pr 올려보겠습니다!